examples/FullExample.ts

```diff
@@ -89,6 +107,7 @@ export default new Workbook({
   namespace: 'employee',
   sheets: {
     Employees,
+    BaseSheet
   },
   portals: [EmployeesPortal],
```

Comment:
@elisadinsmore - I think you'll want to add a `BaseSheetPortal` to this array 

e.g 

`portals: [EmployeesPortal, BaseSheetPortal],`

and then define BaseSheetPortal in a similar way to EmployeesPortal

e.g.

```
const BaseSheetPortal = new Portal({
  name: 'BaseSheetPortal',
  sheet: 'BaseSheet'
})
```

Reply:
Thank you! I didn't actually create a portal for BaseSheet because Linked Fields only work in Workspaces BUT I have now added a new sheet and moved the LinkedField example there, as putting a linkedfield in a deployed portal will break that portal. I've tested deploying my updated example in the example.ts file and everything works now but would still appreciate a second pair of eyes @adambouchard or @paddymul on this.

Reply:
looks good